src/llvm-coverage-instrumentation.md

```diff
@@ -150,50 +151,53 @@ MIR `Statement` into some backend-specific action or instruction.
         match statement.kind {
             ...
             mir::StatementKind::Coverage(box ref coverage) => {
-                self.codegen_coverage(&mut bx, coverage.clone());
+                self.codegen_coverage(&mut bx, coverage.clone(), statement.source_info.scope);
```

Comment:
Do you happen to have a link to the PR changing this? Has it already landed or is the PR still open?

Reply:
Yes, this landed on March 17. @tmiasko added the ability to support inlining with coverage.

rust-lang/rust#83080

See: https://github.com/rust-lang/rust/pull/83080/files#diff-f10b4e4dbb6fc3d26ea86854c45386cf59c2771c75a923f694b4c174e03e0970R114-R115